minichain/p2p.py

```
@@ -1,390 +1,176 @@
+            try: msg = await self.loop.run_in_executor(None, self._to_asyncio.get)
+            except Exception: continue
+            
+            if msg[0] == "MSG":
+                data = msg[1]
+                msg_type, payload = data.get("type"), data.get("data")
+                if msg_type not in SUPPORTED_MESSAGE_TYPES or self._is_duplicate(msg_type, payload): continue
+                self._mark_seen(msg_type, payload)
+                if self._handler_callback: await self._handler_callback(data)
+            elif msg[0] == "PEER_CONNECTED":
+                class MockWriter:
+                    def write(self, data): self.data = data
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `write`

Add return type annotation: `None`

(ANN202)